packages/cfd/src/Components/props.types.ts

```diff
@@ -1,7 +1,7 @@
 import { DetailsOfEachMT5Loginid } from '@deriv/api-types';
 import { TCFDPasswordReset } from '../Containers/props.types';
 
-export type TCFDPlatform = 'dxtrade' | 'mt5';
+export type TCFDPlatform = 'dxtrade' | 'mt5' | 'derivez';
```

Comment:
@hamza-deriv since derivEZ is not released yet, let's add a todo here to update this for derivEZ and CTrader